src/components/vue-form-builder.vue

```diff
@@ -314,6 +319,11 @@ export default {
           .toLowerCase()
           .includes(this.filterQuery.toLowerCase());
       });
+    },
+    containsFormElements() {
```

Comment:
I wonder if this should only check the current page, instead of all pages? If I have multiple empty pages and drag one element onto just one of the empty pages, the "drag an element here" placeholder no longer appears for any of the other empty pages.